pkg/codegen/internal/test/testdata/nested-module-thirdparty/go/foo/deeply/nested/module/init.go

```diff
@@ -32,7 +32,7 @@ func (m *module) Construct(ctx *pulumi.Context, name, typ, urn string) (r pulumi
 }
 
 func init() {
-	version, err := foo - bar.PkgVersion()
```

Comment:
I do no think this previous code compiled.. "foo - bar" is output for identifier "foo-bar". Ouch. 

The reason it passed compile check is that compile check was not deeply compiling ( -v all).

Reply:
Thanks for pointing this out!!!